python_modules/libraries/dagster-dg/dagster_dg/cli/scaffold.py

```diff
@@ -189,6 +196,168 @@ def scaffold_workspace_command(
     scaffold_workspace(name, workspace_config)
 
 
+# ########################
+# ##### GITHUB ACTIONS
+# ########################
+
+
+def _search_for_git_root(path: Path) -> Optional[Path]:
+    if path.joinpath(".git").exists():
+        return path
+    elif path.parent == path:
+        return None
+    else:
+        return _search_for_git_root(path.parent)
+
+
+GITHUB_ACTIONS_WORKFLOW_URL = "https://raw.githubusercontent.com/dagster-io/dagster-cloud-serverless-quickstart/refs/heads/main/.github/workflows/dagster-plus-deploy.yml"
```

Comment:
I think the templates used should be baked in and versioned with the dg release.

This is a supply chain + build reproducibility problem for customers, both from reliability and security - a working dg release may break because of a change, and that a compromise of github or mitm attack can cause issue; and this might be expected to work in an air gap environment (long shot, but eh).

Reply:
Moved to static file packaged with in the python package